src/views/activity/activity.view.tsx

```diff
@@ -86,36 +86,48 @@ const Activity: FC = () => {
           });
         })
         .finally(() => {
-          setAreBridgesDisabled(false);
+          if (isAsyncTaskDataAvailable<Bridge[], undefined, true>(apiBridges)) {
+            getPendingBridges(apiBridges.data)
```

Comment:
Why instead of calling `getPendingBridges` again we just do:

```ts
processFetchBridgesSuccess(apiBridges.data);
```

Reply:
* Because we don't want to call `setLastLoadedItem` or `setApiBridges` (since nothing has changed API-wise)
* Because we need to listen to the callback of `getPendingBridges` to re-enable the bridges after the new pending bridge is listed and `processFetchBridgesSuccess` returns `void`. This is related to fixing the bug:

> Make sure that a deposit can not be finalized twice.